lib/ansible/modules/database/mysql/mysql_db.py

```diff
@@ -23,9 +23,14 @@
   name:
     description:
       - name of the database to add or remove
-      - name=all May only be provided if I(state) is C(dump) or C(import).
+      - I(name=all) May only be provided if I(state) is C(dump) or C(import)
+      - name=db_1,db_2 May only be provided if I(state) is C(dump)
+      - name May only be provided if I(state) is C(dump)
```

Comment:
```suggestion
```

Reply:
@Andersson007: We want to keep both the ways to take a dump.
https://github.com/ansible/ansible/pull/56721#issuecomment-495163838

Reply:
```
      - I(name=db_1,db_2,...) May only be provided if I(state) is C(dump)
```
it is the same as:
```
      - name May only be provided if I(state) is C(dump)
        - db_1
        - db_2
```
moreover, the last one looks very strange.
Even more, IMO, better to use something like "- List of databases is provided with I(state=dump) only" instead of both sentences.
We shouldn't give any examples of possible ways to use lists in DOCUMENTATION block. You show it in EXAMPLES, that's enough

Reply:
Added a single line as narrated in above comment